pkg/third_party/retry/retry_test.go

```diff
@@ -159,8 +157,9 @@ func TestMaxDelay(t *testing.T) {
 	)
 	dur := time.Since(start)
 	assert.Error(t, err)
-	assert.True(t, dur > 170*time.Millisecond, "5 times with maximum delay retry is longer than 170ms")
```

Comment:
https://github.com/avast/retry-go/blob/master/retry_test.go#L264

FYI, refer this.